installer/bundle/bundle_skel.sh

```diff
@@ -625,6 +625,7 @@ cd $EXTRACT_DIR
 # Do we need to remove the package?
 set +e
 if [ "$installMode" = "R" -o "$installMode" = "P" ]; then
+    rm -f /etc/cron.d/OMSConsistencyInvoker
```

Comment:
I see that you use /etc/cron.d many places, can this be converted to a const string? Also is the cron path same for all the distros?